pkgs/dart_mcp/example/workflow_client.dart

```diff
@@ -59,36 +69,38 @@ final argParser =
         abbr: 'v',
         help: 'Enables verbose logging for logs from servers.',
       )
+      ..addOption(
+        'log',
+        abbr: 'l',
+        help:
+            'If specified, will create the given log file and log server '
+            'traffic and diagnostic messages.',
+      )
       ..addFlag('dash', help: 'Use the Dash mascot persona.', defaultsTo: false)
       ..addOption(
         'dtd',
         help: 'Pass the DTD URI to use for this workflow session.',
       )
       ..addOption(
         'model',
-        // defaultsTo: 'gemini-2.0-flash',
-        // defaultsTo: 'gemini-2.5-flash-preview-04-17',
-        defaultsTo: 'gemini-2.5-pro-exp-03-25',
-        // defaultsTo: 'gemini-2.5-flash-preview-04-17',
+        defaultsTo: allowedGeminiModels.first,
+        allowed: allowedGeminiModels,
         help: 'Pass the name of the model to use to run inferences.',
       )
       ..addFlag('help', abbr: 'h', help: 'Print the usage for this command.');
 
 final class WorkflowClient extends MCPClient with RootsSupport {
-  final Logger logger;
-  int totalInputTokens = 0;
-  int totalOutputTokens = 0;
-
   WorkflowClient(
     this.serverCommands, {
     required String geminiApiKey,
+    required String model,
+    required this.logger,
     String? dtdUri,
-    String? model,
     this.verbose = false,
-    required this.logger,
     String? persona,
+    File? logFile,
```

Comment:
We might actually want a file per server? But, this might work too. We can always adjust as needed later if you want to keep just one file for now.